src/index.html

```diff
@@ -7,8 +7,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="title_part_of_sait">
+      <a class="moyo-logo" href="index.html">
+        <img class="logo"
+          src="images/logo.png"
+          alt="Logo shop MOYO"
+        >
+      </a>
+      <nav class="all-button">
+        <ul class="button apple">
+          <a class="apple link_button is-active" href="index.html">Apple</a>
```

Comment:
use BEM, also wrap each link in li tag

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="index.html">Apple</a>
```

Reply:
also no need to name each item differently, add same names to each li and link